src/shogun/preprocessor/RandomFourierGaussPreproc.h

```diff
@@ -16,15 +16,19 @@
 #include <shogun/preprocessor/DensePreprocessor.h>
 
 namespace shogun {
-/** @brief Preprocessor CRandomFourierGaussPreproc
- * implements Random Fourier Features for the Gauss kernel a la Ali Rahimi and Ben Recht Nips2007
- * after preprocessing the features using them in a linear kernel approximates a gaussian kernel
+/** @brief Preprocessor that approximates Gaussian feature map.
  *
- * approximation quality depends on dimension of feature space, NOT on number of data.
+ * Effectively it requires only two parameters for initialization: 
+ * (A) m_kernelwidth: exp(-x^2 / kernel_width)
+ * (B) m_dim_output: Target dimension
+ * 
+ * The random coefficients are stored in the following members
```

Comment:
remove this bit

Reply:
I think you meant removing m_kernelwidth and m_dim_output?. Modified class doc

Reply:
we dont need to state any semantics in the docs. These things outdate and are not useful. You are essentially again describing the code. Just remove them